BookPlayerWatch/LocalPlayback/Player/PlayerManager.swift

```diff
@@ -815,14 +815,35 @@ extension PlayerManager {
       let timePassedLimited = min(max(timePassed, 0), Constants.SmartRewind.threshold)
 
       let delta = timePassedLimited / Constants.SmartRewind.threshold
+      let baseRewindTime = ease(delta) * getMaxInterval()
 
-      // Using a cubic curve to soften the rewind effect for lower values and strengthen it for higher
-      let rewindTime = pow(delta, 3) * Constants.SmartRewind.maxTime
+      // Always rewind by at least minRewind...
+      let rewindTimeMin = max(baseRewindTime, Constants.SmartRewind.minRewind)
 
-      let newPlayerTime = max(CMTimeGetSeconds(self.audioPlayer.currentTime()) - rewindTime, 0)
+      // ...but don't rewind past the beginning of the chapter, or more than the amount of time passed
+      let timeInChapter = item.currentTime - item.currentChapter.start
+      let rewindTimeLimited = min(rewindTimeMin, timeInChapter, timePassed)
```

Comment:
sorry it took me a while to get back to the PR (I was working on #1384), I'm not 100% convinced of the 2 secs minimum rewind and the 60 mins duration update (the rest looks good), but that just could be inertia to new things, I'm running this branch on my phone to see how it feels and report back between sunday and monday